src/extractors/custom/blisterreview.com/index.test.js

```diff
@@ -35,7 +33,7 @@ describe('BlisterreviewComExtractor', () => {
 
       // Update these values with the expected values from
       // the article.
-      assert.equal(title, `2019-2020 Line Outline`);
+      assert.equal(title, `2021-2022 Line Outline`);
```

Comment:
will these values change in the new year?

Reply:
Did we want to add an issue to remind us to look at this in the new year? I can set up a scheduled message on Slack?

Reply:
Sure, that sounds great!